### ISSUE TITLE
Does TWIST lymfoid panel work in scout cancer view? 

### ISSUE THREAD

**Issue by heronikdin** [Clinical-Genomics/scout#2602]

**Comment by dnil** [comment]
I don't think you are missing anything @heronikdin. The view is for SNVs only, but the variant is seemingly annotated and like an SV, which does not display well on that tab. If SVs are to be called for exomes at all, there should instead be an SV file loaded - and you should see an SV button on the case page next to the SNV one. Now the caller may actually have tried to represent just a small indel here - a 5 bp deletion possibly - but doing so in an awkward fashion.

It may be related to the previous similar https://github.com/Clinical-Genomics/BALSAMIC/issues/485, but could just as well also be a new thing; my best guess being a miscommunication between the variant caller and VEP. Do any of you know off the top of your heads @hassanfa or @ashwini06? Otherwise weäll go back to the VCF files and try to piece it together.

**Comment by dnil** [comment]
Hm, doing so seems to say that https://github.com/Clinical-Genomics/BALSAMIC/issues/485 has popped back in again? Or perhaps it was never solved for VarDict, only Sentieon? I´ll transfer this issue over to BALSAMIC for now!

```
2       25475179        .       G       <DEL>   243     PASS    SAMPLE=cleantahr;TYPE=DEL;DP=132;VD=104;AF=0.7879;BIAS=2:2;REFBIAS=71:60;VARBIAS=52:52;PMEAN=72.5;PSTD=1;QUAL=36.3;QSTD=1;SBF=0.5991;ODDRATIO=1.18248;MQ=60;SN=208;HIAF=0.4541;ADJAF=0.7879;SHIFT3=0;MSI=0;MSILEN=0;NM=0.4;HICNT=104;HICOV=229;LSEQ=GGGGCGATGGGGCTGGGGGC;RSEQ=TGCGGGGGGGCGTGGGTAGG;DUPRATE=0;SVTYPE=DEL;SVLEN=7666099;SPLITREAD=13;SPANPAIR=52;CSQ=deletion|
...
```
